release/_releases_json.sh

```diff
@@ -52,7 +52,10 @@ function _process_product {
 		tr -d "/" | \
 		uniq)
 	do
+		if [[ ! "{$product_version}" =~ (7\.[0-9]+\.[0-9]+\-u1(1[3-9]|[2-9][0-9]+|[0-9]{3,})) ]]
```

Comment:
No problem, @pgydev! :slightly_smiling_face: 

I gave it a try and it may be complex

Can you please just check the following?

- Is the product version that we want to avoid, e.g. `u113`, is it following the pattern `7.4.3.113` or `7.4.3.u113`. Can you please double check and make the necessary adjustments if needed?
- In the last part of this regex `[0-9]{3,}`, shouldn't it be `[1-9]{2,}`? Otherwise, we may get numbers like `001`, am I right?

Thanks

Reply:
@natocesarrego ,

- The script opens the https://releases.liferay.com/dxp and there the release `7.4.13-u113` like this and I check this pattern so it won't put it in the json file I tested it. But on the other hand, in the release.properties file under the `target.platform.version` It's like this `7.4.13.u113`.

- I'm not sure, it only catches  `7.*.*-u113` and above. I'm not sure what you mean by getting `001` numbers. 

Thanks :)

Reply:
Thanks for your clarification, @pgydev!